control-plane/catalog/to-consul/resource_test.go

```diff
@@ -2141,7 +2141,7 @@ func createEndpointSlice(t *testing.T, client *fake.Clientset, serviceName strin
 				{
```

Comment:
nit: description of createEndpointSlice says it creates an endpoint slice with 2 endpoints and this now creates 3

Reply:
Fixed